packages/core/src/odata/v4/request-builder/create-request-builder.ts

```diff
@@ -86,7 +87,7 @@ export class CreateRequestBuilder<EntityT extends Entity>
       .then(request => request.execute())
       .then(response =>
         deserializeEntity(
-          response.data.d,
+          extractData(response),
```

Comment:
could you make a second PR, where you look for the many hard coded `.data.d` usage and replace them all.

Additional optimization: Is this something we can know from the system we are calling for example S/4 use `data.d`, C/4 something I do not know, concur XYZ. Perhaps it makes then sense to include this extraction in the builder or so. I do not know.

Reply:
- had a discussion with Marika and decided to make abstract request builders and move the `.data.d` to a single file -> [ticket](https://github.wdf.sap.corp/MA/scp-rocket/issues/1257).
- for different systems, if there is a clear rule, we can use a system-to-extractData mapping. However, even for c/4, the known rule is only for v2 not v4. Also, we decided to also support collection in v2, so SFSF can also make use of the collection type now.